### PR TITLE
[WIP] Try graphviz packmode

### PR DESCRIPTION
**Pull request by mbargull**
Trying out stuff mentioned in https://github.com/regro/cf-scripts/pull/774#issuecomment-579807694

### CODE REVIEW

**Comment by CJ-Wright** [comment]
LGTM! Can you post a screenshot?

**Comment by mbargull** [comment]
Was going to provide examples for each of the commits.
But then GitHub said it doesn't like SVGs... I'll convert and post them later ;).

**Comment by mbargull** [comment]
Picture galore:

gh-774:

3d0843d-python38 svg

----
3d0843d-archrebuild svg

----
----
gh-774 + `packmode=node`:
ae2c876-python38 svg

----
ae2c876-archrebuild svg

----
----
pre-gh-774 + `packmode=node`
7eb4e50-python38 svg

----
7eb4e50-archrebuild svg

----
----
pre-gh-744 + `packmode=array_3`

8322757-python38 svg

----
8322757-archrebuild svg

----
----
gh-744 + `packmode=array_3`

b37b80b5-python38 svg

----
b37b80b5-archrebuild svg

File: conda_forge_tick/auto_tick.py [hunk 1]
```diff
@@ -502,7 +502,7 @@ def migrator_status(
     import graphviz
     from streamz.graph import _clean_text
 
-    gv = graphviz.Digraph(graph_attr={"packmode": "node"})
+    gv = graphviz.Digraph(graph_attr={"packmode": "array_3"})
```
Comment (mbargull) [inline conda_forge_tick/auto_tick.py]: Commit message was supposed to be `Use graphviz packmode=array_3`, of course. (Lazy copy-pasting..)

**Comment by CJ-Wright** [comment]
thoughts @regro/auto-tick ?

**Comment by mariusvniekerk** [comment]
Looks nicer.  I recommend merging

**Pull request merged**